appveyor.yml

```diff
@@ -17,6 +17,14 @@ image: Visual Studio 2019
 matrix:
```

Comment:
@ljharb 
Appveyor offers a WSL `image` too 🤔 
https://www.appveyor.com/docs/windows-images-software/#wsl

Reply:
Trying that here
https://github.com/import-js/eslint-plugin-import/pull/2739